### ISSUE TITLE
All examples crashing when loading layer 24

### ISSUE THREAD

**Issue by timonsku** [mrzl/ofxDarknet#26]
After doing the fixes described here: https://github.com/mrzl/ofxDarknet/issues/23#issuecomment-299926004
I have the issue that every example I'm trying is crashing as soon as it gets to layer 24.
See this screenshot:

![error](http://i.imgur.com/ZHAWEnf.png)

This happening with every example. This specifically is from the yolo2 example.
Same behaviour on Windows 7 and Windows 10 with VS15 and CUDA 8 x64 on 3 different machines.
I got a version of darknet running standalone using this repo: https://github.com/AlexeyAB/darknet

**Comment by naus3a** [comment]
I'm debugging myself the addon and just bumped into this issue: if you run from the console, you'll see that it's just trying to load weight files. When it does not find them, it just explodes :)

~~I'm downloading my weights now and I'll probably test them later tonight or on monday. After that I'll come back here to report if there were further issues.~~

I downloaded the weight files and I confirm that it was only a missing file issue.

**Comment by timonsku** [comment]
Yes I discovered that it tried to load yolo9000.tree with the string "data/yolo9000.tree". Though VisualStudio is setting the working dir to the project folder. not the bin folder. If you run the example manually instead of from VisualStudio it works just fine.
Given that you have your weight file in the data folder.

I did not have to delete any function defintion, all I did was changing that one variable to const and adding the network_kernels.cu to the solution. After that it compiles and runs. Though it behaves very weirdly, every couple of starts it just doesnt detect anything in a scene it detected a ton of things a moment before, sometimes it also looses track of a ton of detected things after a couple of seconds of running. I havent had this issue when running darknet standalone. It is also running extremly slow at about 4-5fps as if it isnt actually running with CUDA.
Performance was pretty much the same with a GTX 680 and a GTX 1080.
